src/main/java/de/zalando/aruha/nakadi/service/subscription/StreamParameters.java

```diff
@@ -0,0 +1,66 @@
+package de.zalando.aruha.nakadi.service.subscription;
+
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+public class StreamParameters {
```

Comment:
Just wanted to point out that there is class https://github.com/zalando/nakadi/blob/nakadi-jvm/src/main/java/de/zalando/aruha/nakadi/service/EventStreamConfig.java
And it has a lot of common properties with your StreamParameters class
Maybe it makes sense to extract a common parent for them.